README.md

```diff
@@ -65,28 +65,50 @@ The plugin can be configured in the [**semantic-release** configuration file](ht
 }
 ```
 
+### Development
+Semantic-release does not need the latest version defined to do its job, to simplify the version management inside your repository `.cabal` file during development time, we strongly recommend using:
```

Comment:
We can explain things a bit more here, like:
``` suggestion
Based on the semantic-release philosophy, you don't need to keep track of the version in your repository (.cabal) anymore. The plugin will build the package with the proper version on the fly, making it simpler for you to maintain your packages. In simpler words, you can change you `.cabal` file to the following:
```

After the `.cabal` file example, we can add some guidance in case people still want to keep track of the version. I was thinking on something like this: 

> If you still want to keep track of the version in the repository, you'll need to allow semantic-release to update your `.cabal` file in your repository after the semantic-release-hackage plugin finishes. This can be achieved with the [@semantic-release/git plugin](https://github.com/semantic-release/git), but keep in mind that semantic-release explicitly [discourages making commits during your release process](https://semantic-release.gitbook.io/semantic-release/support/faq#making-commits-during-the-release-process-adds-significant-complexity).